drivers/storage/portworx/deployment.go

```diff
@@ -1462,6 +1463,7 @@ func (t *template) getCSIVolumeInfoList() []volumeInfo {
 		volumeInfo{
 			name:         "csi-driver-path",
 			hostPath:     t.csiConfig.DriverBasePath(),
+			mountPath:    t.csiConfig.DriverBasePath(),
```

Comment:
as we discussed in the meeting, the kubeletPath should be retrieved from env var.